src/Template/Element/sentences/sentence_and_translations.ctp

```diff
@@ -87,8 +87,8 @@ $sentenceUrl = $this->Url->build([
         <div flex><?= $duplicateWarning ?></div>
     </div>
     <div layout="column">
-        <div layout="row" class="header">
-            <md-subheader flex ng-if="!vm.isMenuExpanded">
+        <div layout="row" layout-align="end" layout-wrap class="header">
+            <md-subheader flex="auto" ng-if="!vm.isMenuExpanded">
```

Comment:
Thank you for the review.

I agree that putting the menu on top is more natural. I also noticed that you removed the `ng-if="!vm.isMenuExpanded"` from the ID+owner element, which is probably for the best when we have another way to make room for it.

However, I don't quite like using the `-xs` attributes to change the layout for small screens. The nice thing about the wrapping-based solution is that it takes the available space into account and switches to two rows exactly when a single row no longer fits.

I think there's a way to have the first element wrap to the second row instead of the last, which would also put the menu on top, but only when necessary. So I'll try that first.